src/frontend/src/pages/CreateProjectPage/CreateProjectFormView.tsx

```diff
@@ -103,19 +102,22 @@ const CreateProjectFormView: React.FC<CreateProjectFormViewProps> = ({ allowSubm
               />
             </FormControl>
           </Grid>
-          <Grid item xs={12} md={3}>
-            <FormControl>
-              <FormLabel>Project Name</FormLabel>
+        </Grid>
+        <Grid container sx={{ marginBottom: '24px' }}>
```

Comment:
this should all be in one grid container, there are styles that/ props that add spacing between parts of a grid. https://mui.com/material-ui/react-grid/